config/src/vespa/config/common/configparser.cpp

```diff
@@ -156,9 +156,9 @@ ConfigParser::stripLinesForKey(std::string_view key,
                                std::set<vespalib::string>& config)
 {
     vespalib::string value;
-    for (std::set<vespalib::string>::iterator it = config.begin(); it != config.end();) {
+    for (auto it = config.begin(); it != config.end();) {
         if (getValueForKey(key, *it, value)) {
-            std::set<vespalib::string>::iterator it2 = it++;
+            auto it2 = it++;
```

Comment:
This line and the next can probably be combined to something like `it = config.erase(it);`

Reply:
Fixed